Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/Components/ReelThumbnailController.cs

```diff
@@ -49,7 +51,10 @@ private async UniTaskVoid LoadImageAsync(CancellationToken token)
             view.loadingBrightView.StartLoadingAnimation(view.thumbnailImage.gameObject);
 
             Texture2D thumbnailTexture = await cameraReelScreenshotsStorage.GetScreenshotThumbnailAsync(CameraReelResponse.thumbnailUrl, token);
-            view.thumbnailImage.sprite = Sprite.Create(thumbnailTexture, new Rect(0, 0, thumbnailTexture.width, thumbnailTexture.height), Vector2.zero);
+            float originalToSmallerRatio = thumbnailTexture.height * 1f / rectTransform.rect.height;
```

Comment:
yes, the intention is to cast the type. As far as I knew multiplying by 1f was "faster" than explicitly casting to float an int. Or are you telling me this to make it more readable?

Reply:
I've never heard about that optimization, I will investigate because I like this kind of tricks, although we are talking about nanoseconds. Nevermind.

Reply:
sure, especially in this case it could probably be considered negligible. Just wanted to make sure that it wasn't a readability thing